sal_interfaces/FiberSpectrograph/FiberSpectrograph_Telemetry.xml

```diff
@@ -1,39 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
-<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
-<SALTelemetry>
-    <Subsystem>FiberSpectrograph</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>FiberSpectrograph_spectTemperature</EFDB_Topic>
-    <Description>Temperature of FiberSpectrograph</Description>
-    <item>
-        <EFDB_Name>temperature</EFDB_Name>
-        <Description>temperature in deg_C</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>deg</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>timestamp</EFDB_Name>
-        <Description>timestamp</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-</SALTelemetry>
-<SALTelemetry>
-    <Subsystem>FiberSpectrograph</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>FiberSpectrograph_timestamp</EFDB_Topic>
-    <item>
-        <EFDB_Name>timestamp</EFDB_Name>
-        <Description>timestamp</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-</SALTelemetry>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+    <SALTelemetry>
+        <Subsystem>FiberSpectrograph</Subsystem>
+        <Version>3.7.1</Version>
+        <Author/>
+        <EFDB_Topic>FiberSpectrograph_temperature</EFDB_Topic>
+        <Description>Temperature of the FiberSpectrograph</Description>
+        <item>
+            <EFDB_Name>temperature</EFDB_Name>
+            <Description>Measured internal spectrograph temperature in degrees Celcius.</Description>
```

Comment:
I suggest dropping "in degrees Celsius" because it duplicates information in the units -- here and elsewhere